packages/contracts/test/contracts/OVM/bridge/base/OVM_L1CrossDomainMessenger.spec.ts

```diff
@@ -370,9 +377,84 @@ describe('OVM_L1CrossDomainMessenger', () => {
           0,
           proof
         )
-      ).to.be.revertedWith(
-        'Only OVM_L2MessageRelayer can relay L2-to-L1 messages.'
-      )
+      ).to.be.revertedWith('Pausable: paused')
+    })
+
+    describe('blockMessage and allowMessage', () => {
+      it('should revert if called by an account other than the owner', async () => {
```

Comment:
This test case fits a lot into it:
- block
- test for failure
- allow 
- test for success

But it felt cleaner to keep the round trip together.